cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -519,9 +519,9 @@ def test_update_from_json(self, store, send_signal, tracker, uuid):
         uuid.return_value = "mockUUID"
         self.course = CourseFactory.create(default_store=store)
         test_grader = CourseGradingModel.fetch(self.course.id)
+        # there should be no event raised after this call, since nothing got modified
```

Comment:
could you add some assertions that actually measure that there were no events fired / that that event wasn't fired?

Reply:
i know I did that in the teams csv tests so you could look there

Reply:
The examples i found in test_csv don't quite apply here. This comment is more of a guidance to myself. The actual assert happens below, along line 550. If you compare it with the previous version, notice that here was an even fired anytime `update_from_json` was called. This not the case anymore.

Reply:
Ah, alright yeah I see it now!